### PR TITLE
Self Referee section

### PR DESCRIPTION
**Pull request by devianyeth**

### CODE REVIEW

File: src/components/SelfReferee/index.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+//nextjs Functional component
+
+import React from "react";
+import Button from "../UI/Button";
+import Content from "../UI/Content";
+import styles from "./SelfReferee.module.css";
+
+const SelfRefree = () => {
+  return (
+    <>
```
Comment (NakedFool) [inline src/components/SelfReferee/index.tsx:10]: we shouldn't wrap <Content/> here :)

File: src/components/SelfReferee/index.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      <Content title="Self-Refereeing">
```
Comment (NakedFool) [inline src/components/SelfReferee/index.tsx:11]: we can add [work in progress] here

File: src/components/SelfReferee/index.tsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+          <br />
+          <br />
+          <p>
+            If you hold an nft, you can send a transaction attesting to a
+            submitted scorecard that conveys correct results of off-chain
+            events.
+          </p>
+          <p className={styles.attestationConfirm}>
+            You’ve attested to scorecard with root <b>0xasdf...1234</b> which
+            matches these results.
```
Comment (NakedFool) [inline src/components/SelfReferee/index.tsx:33]: **these results** has an underline, you can wrap it with a span! :)

File: src/components/SelfReferee/index.tsx [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+          <p>
+            Each game phase must also be queued by someone in the public in a
+            timely manner.
+          </p>
+          <Button onClick={() => {}} size="medium">
+            QUEUE PHASE 4
+          </Button>
+          <br />
+          <br />
+          <Button onClick={() => {}} size="big" color="#736B6F">
```
Comment (NakedFool) [inline src/components/SelfReferee/index.tsx:49]: you can remove this button for now,  it is ok if we show only Queue button above for now

**Review by NakedFool — CHANGES_REQUESTED** [review]
Very well done @devianyeth !

Just a few minor changes and we are good to push 🍻

File: src/components/SelfReferee/index.tsx (hunk 3 above)
Comment (NakedFool) [reply src/components/SelfReferee/index.tsx:33]: One more thing, all paragraphs in self-referee section should have `font-weight: 500`

**Comment by devianyeth** [comment]
@NakedFool, I've made the changes

**Pull request merged**